dev/deps/spark-deps-hadoop-3.2

```diff
@@ -5,6 +5,7 @@ ST4-4.0.4.jar
 accessors-smart-1.2.jar
 activation-1.1.1.jar
 aircompressor-0.10.jar
+ant-1.6.5.jar
```

Comment:
Dang, we pull in all of Ant now? OK if it's required

Reply:
I'm not sure do we really need it. It comes from `tomcat:jasper-compiler:jar:5.5.23`: 
```
[INFO] +- org.apache.hive:hive-serde:jar:2.3.4:compile
[INFO] |  +- org.apache.hive:hive-service-rpc:jar:2.3.4:compile
[INFO] |  |  +- tomcat:jasper-compiler:jar:5.5.23:compile
[INFO] |  |  |  +- javax.servlet:jsp-api:jar:2.0:compile
[INFO] |  |  |  \- ant:ant:jar:1.6.5:compile
[INFO] |  |  \- tomcat:jasper-runtime:jar:5.5.23:compile
[INFO] |  |     +- javax.servlet:servlet-api:jar:2.4:compile
[INFO] |  |     \- commons-el:commons-el:jar:1.0:compile
```

Reply:
Yeah I'm surprised if we need jasper at all. That's a JSP compiler.

Reply:
It seem don't need jasper. I used to exclude jasper to run all the tests..
https://github.com/apache/spark/pull/24044/files#diff-600376dffeb79835ede4a0b285078036R1761

Reply:
We don't need this: [HIVE-19799](https://issues.apache.org/jira/browse/HIVE-19799) remove jasper dependency